NCP/openncp-national-connector/src/main/java/dk/openncp/nationalconnector/fhir/DenmarkDispatchingService.java

```diff
@@ -0,0 +1,19 @@
+package dk.openncp.nationalconnector.fhir;
+
+import eu.europa.ec.sante.openncp.core.common.fhir.context.EuRequestDetails;
+import eu.europa.ec.sante.openncp.core.common.fhir.services.DispatchingService;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DenmarkDispatchingService implements DispatchingService {
+    @Override
+    public <T extends IBaseResource> T dispatchSearch(EuRequestDetails euRequestDetails, String s) {
+        return null;
```

Comment:
I assume this is a placeholder, is the burden on the NCP implementation to detect this as an error, or should we throw an exception instead?

Reply:
It's probably better to throw an exception here, I'll do that.